packages/lib/event-types/transformers/api-response.ts

```diff
@@ -189,4 +201,63 @@ function getResponseEventTypeBookingFields(
   });
 }
 
-export { getResponseEventTypeLocations, getResponseEventTypeBookingFields };
+function getResponseEventTypeIntervalLimits(
```

Comment:
Would be good to add tests of functions added to `api-response.ts` in `api-response.spec.ts` - it should be fast to add and would be good because transformation logic can sometimes be hard to grasp right away.

Reply:
added https://github.com/calcom/cal.com/pull/16128/commits/b8bf222e894147dc9d0fc055443188cecc25cf29

Reply:
Amazing!